src/boot/app/app-direct-exports.ts

```diff
@@ -103,4 +103,4 @@ export { AppContextProvider } from './app-context-provider';
 
 export type { AuthGuardProps } from '../../ui-extras/auth-guard';
 export { AuthGuard } from '../../ui-extras/auth-guard';
-export { soapFetchV2 } from '../../network/fetch';
+export { soapFetchV2 } from '@zextras/carbonio-ui-soap-lib';
```

Comment:
suggestion: noone use soapFetchV2 at the moment cause it is introducted recently....can we remove it direcly avoiding someone start to use it?
Ok it is a BC but I think is acceptable